### PR TITLE
Add German localization

### PR DESCRIPTION
**Pull request by tkrajacic**
A first stab at a German localization

### CODE REVIEW

File: src/i18n/de.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        start: 'Jetzt loslegen',
+        join: 'Am Chat teilnehmen'
+    },
+    nav: {
+        code: 'Code',
+        cloud: 'Cloud',
+        docs: 'Doku'
+    },
+    bio: {
+        first: 'Asynchrone, Ereignis-gesteuerte Architektur die auf Apples <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO</a> basiert und <a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">höchste Performance</a> liefert.',
```
Comment (code28) [inline src/i18n/de.js]: I understand, that `Ereignis-gesteuerte` sets a focus on the `Ereignis`-Scheme, but I think this word looks weird. So I'd rather use the english `event`, so `Event-gesteuerte` or `ereignisgesteuerte`.

Comment (code28) [reply src/i18n/de.js]: And there is a comma missing: `...Architektur, die...`

File: src/i18n/de.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        second: 'In <a target="_blank" href="https://swift.org">Swift</a> geschrieben, der Sprache, die nicht nur performant, sondern auch einfach zu erlernen ist.',
+        third: 'Ausdrucksstarkes, <a target="_blank" href="https://developer.apple.com/videos/play/wwdc2015/408/">Protokollorientiertes Design</a> mit Fokus auf Typsicherheit und einfach zu wartendem Code.'
```
Comment (code28) [inline src/i18n/de.js]: Since `protokollorientiertes` is an adjective, it shouldn't have a capital P.

File: src/i18n/de.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        options: {
+            code: {
+                description: 'Schlage Features vor, melde Bugs, und integriere deinen Code auf GitHub!'
+            },
+            chat: {
+                title: 'Am Chat teilnehmen',
+                description: 'Hilf Neulingen beim Einstieg, und starte mit Vapor.'
+            },
+            donate: {
+                title: 'Werde Unterstützer',
```
Comment (code28) [inline src/i18n/de.js]: I'd suggest to keep it in the same style as `Am Chat teilnehmen`, so: `Unterstützer werden`.

File: src/i18n/de.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                description: 'Unterstütze die Entwicklung mit einer einmaligen oder wiederkehrenden Spende.'
+            }
+        }
+    },
+    sponsors: {
+        title: 'Unsere Sponsoren'
+    },
+    footer: {
+        contact: 'Kontakt',
+        press: 'Presse Unterlagen',
```
Comment (code28) [inline src/i18n/de.js]: Please no whitespace there! So either `Presseunterlagen` or just `Presse`.

**Review by code28 — CHANGES_REQUESTED** [review]

File: src/i18n/de.js (hunk 1 above)
Comment (tkrajacic) [reply src/i18n/de.js]: I went with `Event-gesteuerte` for now, but I think both your suggestions are very good.

File: src/i18n/de.js (hunk 2 above)
Comment (tkrajacic) [reply src/i18n/de.js]: Absolutely! Fixed.

File: src/i18n/de.js (hunk 3 above)
Comment (tkrajacic) [reply src/i18n/de.js]: Good suggestion! Fixed.

File: src/i18n/de.js (hunk 4 above)
Comment (tkrajacic) [reply src/i18n/de.js]: I went with just `Presse` for now.

**Review by code28 — APPROVED** [review]
👌

**Review by tanner0101 — CHANGES_REQUESTED** [review]
This looks great, thanks. And since we have a third party approval, we can merge. Can you just add the `de` entry to [`index.js`](https://github.com/vapor/website/blob/master/src/i18n/index.js#L7)?

**Review by tanner0101 — APPROVED** [review]

**Comment by penny-coin** [comment]
Hey @tkrajacic, you just merged a pull request, have a coin! 

You now have 19 coins.

**Comment by tanner0101** [comment]
~~Thanks~~ danke :)

**Comment by tanner0101** [comment]
screen shot 2019-02-26 at 7 16 02 pm

**Pull request merged**